### DISCUSSION TITLE
[SOLVED] Comment about Save-Data header.

### DISCUSSION THREAD

**Discussion by bat999** [bromite/bromite#1868] [General]
Hi
In Bromite version 99...

I suppose "data saver header" patch has been removed because of CHANGELOG entry "complete disabling of client hint headers (thanks to @uazo)".
And Save-Data is a client hint request header.

**Comment by csagan5** [comment]
It was removed by mistake as I thought it was not relevant anymore; I will re-introduce it, this header is not considered a client hint within Chromium so it will not be affected.

**Reply by bat999** [reply]
Fixed in Bromite v99.0.4844.58.
Thanks.

There he is.
Last man standing.
:-)

![save_data_header](https://user-images.githubusercontent.com/640761/158034249-31134371-6e0c-4e24-940d-4f4e1f7e4949.jpg)
